peps/pep-0771.rst

```diff
@@ -6,6 +6,8 @@ Status: Draft
 Type: Standards Track
 Topic: Packaging
 Created: 13-Jan-2025
+Post-History:
+  `15-Jan-2025 <https://discuss.python.org/t/77892/>`__,
```

Comment:
Do we want to include the topic to be created now here, just before merging?

Reply:
If rather do it the other way around if possible, that is do a follow up small PR with the thread otherwise if I make the new thread now it will temporarily link to the old version of the PEP and could cause confusion?

Reply:
Or is there a way to reserve a thread URL before actually making it public?

Reply:
> temporarily link to the old version of the PEP

For no more than a minute, though.

> reserve a thread URL before actually making it public?

You can create a thread with only a title but no text, and edit the text in later. But this is probably overkill. If you're really worried, you can create the thread with the [preview link](https://pep-previews--4238.org.readthedocs.build/) and edit it to https://peps.python.org/pep-0771/ the moment this is merged?

A

Reply:
> For no more than a minute, though.

Sorry maybe there's a subtlety that I don't understand, but this will rely on you or someone else merging the PR as soon as I add the commit here to add the link to the thread, so in principle could be a much longer time? Or are you saying you are ready to merge at short notice? 😆